app/src/componentLibrary/sidebar/sidebar.jsx

```diff
@@ -66,7 +70,7 @@ export const Sidebar = ({
       <aside className={cx('sidebar')}>
         {logoBlock}
         <div className={cx('main-block-wrapper')}>
-          {createMainBlock(onOpenSidebar, onCloseSidebar)}
+          {createMainBlock(onOpenSidebar, onCloseSidebar, getIsSidebarCollapsed)}
```

Comment:
Why not just use the `isOpenSidebar` instead?

Reply:
isOpenSidebar is always set to true on the onClick event, as it is also set to true on onMouseEnter. 
The sidebar expands due to the CSS transition delay

Reply:
Ok, got it.